wow_library/include/Attributes.hpp

```diff
@@ -124,6 +127,7 @@ struct Special_stats
     double haste{};
     double damage_mod_physical{};
     double stat_multiplier{};
+    double str_multiplier{};
```

Comment:
nit: should be ordered as in constructor

Reply:
Really appreciate these style comments! Also removed a line break between armor_pen and ap_multiplier that didn't look like it made sense. If there's a reason for that to be there I'd be curious to hear it.